moveit_ros/moveit_servo/include/moveit_servo/servo.hpp

```diff
@@ -122,14 +128,19 @@ class Servo
    * \brief Get the current state of the robot as given by planning scene monitor.
    * @return The current state of the robot.
    */
-  KinematicState getCurrentRobotState() const;
+  KinematicState getCurrentRobotState();
```

Comment:
Why is this not `const` if it just gets the robot state variable?

This is impacting the constness of a bunch of other functions as well